### PR TITLE
Update transfers csv columns

### PR DESCRIPTION
**Pull request by zhinfuhnya**
https://www.pivotaltracker.com/n/projects/2167210/stories/178562909

### CODE REVIEW

File: app/decorators/award_decorator.rb [hunk 1]
```diff
@@ -75,20 +77,28 @@ def to_csv # rubocop:todo Metrics/CyclomaticComplexity
...
+      account&.email,
       recipient_wallet&.address,
       account&.decorate&.verification_state,
       issuer.managed_account_id || issuer.id,
       issuer.first_name,
       issuer.last_name,
       latest_blockchain_transaction&.source,
       total_amount,
-      ethereum_transaction_address,
+      transaction_hash,
```
Comment (Riddlerrr) [inline app/decorators/award_decorator.rb:88]: is can be just: `latest_blockchain_transaction&. tx_hash`

**Review by Riddlerrr — CHANGES_REQUESTED** [review]

File: app/decorators/award_decorator.rb [hunk 2]
```diff
@@ -63,27 +64,32 @@ def to_csv_header
...
+      account&.email,
       recipient_wallet&.address,
       account&.decorate&.verification_state,
       issuer.managed_account_id || issuer.id,
       issuer.first_name,
       issuer.last_name,
       latest_blockchain_transaction&.source,
       total_amount,
-      ethereum_transaction_address,
+      paid? ? ethereum_transaction_address : nil,
```
Comment (Riddlerrr) [inline app/decorators/award_decorator.rb:90]: I think it's ok to use just `ethereum_transaction_address` without conditions

**Review by Riddlerrr — APPROVED** [review]

File: app/decorators/award_decorator.rb (hunk 2 above)
Comment (zhinfuhnya) [reply app/decorators/award_decorator.rb:90]: @Riddlerrr regarding to requirements need display address only if transfer is paid.

Comment (Alexey1100) [reply app/decorators/award_decorator.rb:90]: Yep, otherwise it would be displayed for failed tx as well

File: app/decorators/award_decorator.rb [hunk 3]
```diff
@@ -63,27 +64,32 @@ def to_csv_header
...
+  # rubocop:todo Metrics/CyclomaticComplexity
+  # rubocop:todo Metrics/PerceivedComplexity
+  def to_csv
     [
       id,
       transfer_type&.name,
       account&.managed_account_id || account&.id,
       account&.first_name,
       account&.last_name,
+      account&.email,
```
Comment (Alexey1100) [inline app/decorators/award_decorator.rb:82]: There's an edge case when award is issued to an email address, without an account.

I think it should be:

```ruby
account&.email || email
```

**Review by Alexey1100 — CHANGES_REQUESTED** [review]

**Review by Alexey1100 — APPROVED** [review]

**Pull request merged**